CHANGELOG.md

```diff
@@ -15,15 +41,24 @@
   - `quay.io/signalfx/splunk-otel-collector-windows:<version>`
   - `quay.io/signalfx/splunk-otel-collector-windows:<version>-2019`
   - `quay.io/signalfx/splunk-otel-collector-windows:<version>-2022`
-
```

Comment:
Please re-add this line break.  Otherwise there won't be a break with following line.

Reply:
added back.